pkg/pluginmanager/manager.go

```diff
@@ -664,6 +627,59 @@ func InstallPluginsFromGroup(pluginName, groupIDAndVersion string, options ...Pl
 	return groupIDAndVersion, nil
 }
 
+// GetPluginsFromGroup fetches the plugins from the specified plugin group id and version.
+func GetPluginsFromGroup(groupIDAndVersion string, options ...PluginManagerOptions) (*plugininventory.PluginGroup, *plugininventory.PluginGroupIdentifier, error) {
```

Comment:
Could you rename this function.  Although it is being used to retrieve the plugin list it does not actually _return_ a list of plugins, but returns the plugin group itself, from an name.
You can just name it `GetPluginGroup`